logistics_project/apps/malawi/warehouse/models.py

```diff
@@ -46,7 +46,14 @@ class ProductAvailabilityData(MalawiWarehouseModel):
     managed_and_without_stock = models.PositiveIntegerField(default=0)
     managed_and_without_data = models.PositiveIntegerField(default=0)
     managed_and_emergency_stock = models.PositiveIntegerField(default=0)
-    
+
+    def __unicode__(self):
+        return '{0}:{1} ({2}) total: {3} managed: {4} with_stock: {5}, stockout: {6}'.format(
```

Comment:
heh, this unicode function is returning bytes (unless one of the format parameters is unicode)